src/renderer/widgets/CreateWallet/model/form-model.ts

```diff
@@ -0,0 +1,113 @@
+import { createForm } from 'effector-forms';
+import { combine, createEvent } from 'effector';
+import { sample } from 'lodash';
+
+import { Chain, CryptoType, MultisigAccount } from '@shared/core';
+import chains from '@shared/config/chains/chains.json';
+import { accountUtils, walletModel, walletUtils } from '@entities/wallet';
+import { networkUtils } from '@entities/network';
+import { FormParams } from '../lib/types';
+import { signatoryModel } from './signatory-model';
+import { toAccountId } from '@shared/lib/utils';
+
+const $createMultisigForm = createForm<FormParams>({
+  fields: {
+    threshold: {
+      init: 0,
+      rules: [
+        {
+          name: 'moreOrEqualToTwo',
+          validator: (threshold) => threshold >= 2,
+        },
+      ],
+    },
+    chain: {
+      init: chains[0] as Chain,
+    },
+    name: {
+      init: '',
+      rules: [
+        {
+          name: 'notEmpty',
+          validator: (name) => name !== '',
+          errorText: 'createMultisigAccount.disabledError.emptyName',
+        },
+      ],
+    },
+  },
+  validateOn: ['submit'],
+});
+
+const formSubmitted = createEvent<void>();
```

Comment:
Native event to call after form is submitted - `$form.formValidated`
It doesn't cover your case?

Reply:
oh, I didn't know about this one. I changed it.